test/schematic-line-map.js

```diff
@@ -1,200 +0,0 @@
-/** A simple schematic map of all of the trips in a Transitive.js dataset */
```

Comment:
The schematic line map is another useful visualization that should not be removed outright.